### PR TITLE
wob: init

### PR DESCRIPTION
**Pull request by mich-adams**
adds a module for wob colors

### CODE REVIEW

File: modules/wob/hm.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  config = lib.mkIf (config.stylix.enable && config.stylix.targets.wob.enable) {
+    services.wob.settings = {
+      "" = {
+	border_color = config.lib.stylix.colors.base05;
+	background_color = config.lib.stylix.colors.base00;
+	bar_color = config.lib.stylix.colors.base0A;
+	overflow_bar_color = config.lib.stylix.colors.base08;
+	overflow_background_color = config.lib.stylix.colors.base00;
+	overflow_border_color = config.lib.stylix.colors.base05;
```
Comment (trueNAHO) [inline modules/wob/hm.nix:15]: Could you replace the tab indenations with spaces, as this is the current Stylix convention?

File: modules/wob/hm.nix [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      };
+    };
+  };
+}
+
+
```
Comment (trueNAHO) [inline modules/wob/hm.nix:21]: Consider removing these empty lines.

**Review by mich-adams — COMMENTED** [review]
Thanks, replaced the tabs with spaces.

**Review by danth — APPROVED** [review]
LGTM; have not tested.

**Pull request merged**